test/simapp/export.go

```diff
@@ -17,9 +19,10 @@ import (
 // file.
 func (app *SimApp) ExportAppStateAndValidators(
 	forZeroHeight bool, jailAllowedAddrs []string,
+	modulesToExport []string,
 ) (servertypes.ExportedApp, error) {
 	// as if they could withdraw from the start of the next block
-	ctx := app.NewContext(true, tmproto.Header{Height: app.LastBlockHeight()})
```

Comment:
(https://github.com/cosmos/cosmos-sdk/pull/16342) NewContext was renamed to NewContextLegacy